### PR TITLE
runtime: make gc and scheduler asserts/logging settable with build tags

### PR DESCRIPTION
**Pull request by dgryski**

### CODE REVIEW

File: src/runtime/build_asserts.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//go:build runtime_asserts
+
+package runtime
+
+// enable assertions for the garbage collector
+const gcAsserts = true
+
+// enable asserts for the schedule:w
```
Comment (aykevl) [inline src/runtime/build_asserts.go]: Looks like you're using vim? ;)

**Review by aykevl — COMMENTED** [review]
LGTM with a minor nit

File: src/runtime/build_asserts.go (hunk 1 above)
Comment (dgryski) [reply src/runtime/build_asserts.go]: :D. Fixed.

**Review by aykevl — APPROVED** [review]

**Comment by deadprogram** [comment]
@dgryski seems like something might be wrong with submodule? The fail on CircleCI is a bit weird.

**Comment by dgryski** [comment]
Yeah, something's up with Circle CI.  I think the setup script doesn't like the fact that the commit is on a fork and not a branch or something.  I think there's a larger problem going on here, but not one that was introduced with this PR..

**Comment by deadprogram** [comment]
OK, then let's merge.

**Pull request merged**